packages/workflow/src/Extensions/DateExtensions.ts

```diff
@@ -317,36 +421,150 @@ isInLast.doc = {
 
 toDateTime.doc = {
 	name: 'toDateTime',
-	description: 'Convert a JavaScript Date to a Luxon DateTime.',
+	description:
+		'Converts a JavaScript Date to a Luxon DateTime. The DateTime contains the same information, but is easier to manipulate.',
+	examples: [
+		{
+			example: "jsDate = new Date('2024-03-30T18:49')\njsDate.toDateTime().plus(5, 'days')",
```

Comment:
```suggestion
			example: "const jsDate = new Date('2024-03-30T18:49')\njsDate.toDateTime().plus(5, 'days')",
```

Reply:
Would like to keep the original format here, it's what is used for all examples, defining variables in expressions is not possible anyway